fw/ja5_conf.c

```diff
@@ -0,0 +1,176 @@
+/**
+ *		Tempesta FW
+ *
+ * Transport Layer Security (TLS) interfaces to Tempesta TLS.
+ *
+ * Copyright (C) 2015-2024 Tempesta Technologies, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify it
```

Comment:
Please also update `etc/tempesta_fw.conf` with an example of the configuration. Also don't forget to add the configuration information to https://tempesta-tech.com/knowledge-base/HTTP-security/ or https://tempesta-tech.com/knowledge-base/Handling-clients/ or a new page

Reply:
Added doc to etc/tempesta_fw.conf